app/views/certificates/_certificate.html.erb

```diff
@@ -0,0 +1,44 @@
+<style>
+
+  .mu-certificate-box {
+    overflow: hidden;
+    height: 210mm;
+    width: 297mm;
+    max-height: 210mm;
+    max-width: 297mm;
+    min-height: 210mm;
+    min-width: 297mm;
+    margin: 0;
+    box-sizing: border-box;
+    position: relative;
+    border: 1px solid lightgrey;
+  }
+
+  .mu-certificate-box section {
+    background: transparent;
+    z-index: 10;
+  }
+
+  .background-image {
+    position: absolute;
+    height: 210mm;
+    width: 297mm;
+    margin: 0;
+    padding: 0;
+    z-index: 0;
+  }
+
+  .qr-code {
+    position: absolute;
+    z-index: 10;
+  }
+
+</style>
+
+<div class="mu-certificate-box">
+  <img src="<%= certificate.background_image_url %>" class="background-image"/>
+  <%= ERB.new(certificate.template_html_erb).result(certificate.locals.instance_eval { binding }).html_safe %>
```

Comment:
Just curious: do we need to manually create the ERB in and erb file? Can we just use something like `render`?

Reply:
I don't know that render can take inline option...